lib/active_fedora/fixity_service.rb

```diff
@@ -32,10 +32,7 @@ def fixity_graph
       ::RDF::Graph.new << ::RDF::Reader.for(:ttl).new(response.body)
     end
 
-    def status_url
-      ::RDF::URI("http://fedora.info/definitions/v4/repository#status")
```

Comment:
I can't see a case where you'd want to override this. Fcrepo will always use this predicate. I can't see this code being repurposed for anything else.